test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+using namespace System.Management.Automation.Internal
+
+Describe 'Null Representatives' -Tags 'CI' {
+
+    Context 'Comparisons with $null' {
```

Comment:
although not specifically mentioned or explicitly tested, I assume that:

```powershell
if ( -not [dbnull]::Value ) {
   "nothing up my sleeves"
}
```
will work as expected and emit `nothing up my sleeves`

Reply:
Yep! `-not $var` forces a conversion to Boolean if the object isn't already a Boolean value.

I can add an explicit test if you feel it's warranted?

Reply:
Done! 🙂